frontend/packages/knative-plugin/src/components/overview/EventPubSubSubscribers.tsx

```diff
@@ -0,0 +1,102 @@
+import * as React from 'react';
+import { OverviewItem } from '@console/shared';
+import { referenceFor } from '@console/internal/module/k8s';
+import { SidebarSectionHeading, ResourceLink } from '@console/internal/components/utils';
+import { Button, Grid, GridItem } from '@patternfly/react-core';
+import { AngleDownIcon, AngleRightIcon } from '@patternfly/react-icons';
+import { Subscriber } from '../../topology/knative-topology-utils';
+import './EventPubSubSubscribers.scss';
+import FilterTable from './FilterTable';
+
+type PubSubSubscribersProps = {
+  item: OverviewItem & {
+    subscribers?: Subscriber[];
+  };
+};
+
+export const PubSubSubscribers: React.FC<PubSubSubscribersProps> = ({ item }) => {
```

Comment:
Just a thought, should we be just sending the subscribers here instead of full item object? WDYT?